### DISCUSSION TITLE
Base path - Did not match on client and server

### DISCUSSION THREAD

**Discussion by Duckinm** [blitz-js/blitz#2590] [Q&A]
I got issues when I added basepath and using import { Link } from 'blitz'.

As I try using it with many approaches I still got a warning on the client and server to render the different paths. so I decide to change all of it to next/link

Is there any other alternative way to working with basepath?

**Comment by flybayer** [comment]
Sounds like a bug. Can you open an issue with steps to reproduce?

**Reply by Duckinm** [reply]
#2610

Done!
